### DISCUSSION TITLE
[Enhancement]: Topic operator point to external managed kafka

### DISCUSSION THREAD

**Discussion by rlanhellas** [strimzi/strimzi-kafka-operator#10620] [General]
### Related problem

Today the Topic Operator, consequently the CRD, just have the option to manage strimzi kafka. I would like to keep using my external kafka managed like AWS MSK but use strimzi for everything else like topic management 

### Suggested solution

Create a property or label called bootstrap.servers instead of cluster label only and change the operator to use it

**Comment by scholzj** [comment]
You can connect it to any Kafka cluster as long as it supports some common authentication mechanism. You can use the standalone installation for it.

**Reply by rlanhellas** [reply]
Nice. Do you have any documentation link or an example ? Because I really didn't find it.

**Reply by scholzj** [reply]
There are some docs about the standalone Topic Operator. I do not have a link at hand, but if you search for _standalone_ in the docs you should find it. But there is no guide specifically for Amazon MSK or anythign like that.

**Reply by fvaleri** [reply]
Hi @rlanhellas, you can [look here](https://strimzi.io/docs/operators/latest/deploying#deploying-the-topic-operator-standalone-str) (procedure, point 4).

**Reply by rlanhellas** [reply]
thank you will try it out
